app/code/Magento/Rss/Model/Rss.php

```diff
@@ -92,7 +102,13 @@ public function setDataProvider(DataProviderInterface $dataProvider)
      */
     public function createRssXml()
     {
-        $rssFeedFromArray = \Zend_Feed::importArray($this->getFeeds(), 'rss');
-        return $rssFeedFromArray->saveXML();
+        $feed = $this->feedFactory->create(
+            $this->getFeeds(), 
+            FeedFactoryInterface::DEFAULT_FORMAT
```

Comment:
Currently, you are referencing undefined constant, as you changed the name of this constant in FeedFactoryInterface

That's why you have Red Unit and Static tests:
```
There was 1 error:
1) Magento\Rss\Test\Unit\Model\RssTest::testCreateRssXml
Error: Undefined class constant 'DEFAULT_FORMAT'
/home/travis/build/magento/magento2/app/code/Magento/Rss/Test/Unit/Model/RssTest.php:156
```